### ISSUE TITLE
how to build opsu?

### ISSUE THREAD

**Issue by snowythebudg** [itdelatrisu/opsu#647]
hello. i would like to build opsu! and make a modified version of it. i've read the license and i was just wondering, do i have your permission to make a version of opsu!, i'll remove the branding just for your sake and legal sakes but how do i build it. ive never heard of maven or gradle in my life

**Comment by Cefaxion** [comment]
I am not a lawyer but seeing how abandoned it is, no one would care if they modified it

**Comment by snowythebudg** [comment]
yeah but how do you build opsu. i have no clue what maven or gradle is

**Comment by yugecin** [comment]
Maven and Gradle are build automation software, there are links in the readme.
It doesn't seem like there's still a maven configuration, so that's not going to work despite the readme saying so.
The gradle configuration is still there, so you can use that.
So to build, this command should work to run:
```
gradlew.bat run
```
and to make a jar:
```
gradlew.bat jar
```
> This will compile a jar to `build/libs/opsu-${version}.jar` with the libraries, resources and natives packed inside the jar. 

if you're using linux, do `./gradlew` instead of `gradlew.bat`

**Comment by snowythebudg** [comment]
so where would i run these commands. and im assuming i need the source code
